### DISCUSSION TITLE
Incorrect glyphs displayed

### DISCUSSION THREAD

**Discussion by twiclo** [kovidgoyal/kitty#7880] [Q&A] [answered]
Everywhere on my system, not just in kitty, I'm seeing the wrong glyphs displayed. Here's one example character: 
On my system that look like this:
![image](https://github.com/user-attachments/assets/0460e252-91c1-48b6-8434-9fec05512dff)
But when I paste that character into the nerd fonts website it shows a folder, which is what it's supposed to be:
![image](https://github.com/user-attachments/assets/6ec47b2a-6824-4d1a-8655-a358f39f4b1a)

I know this issue may span beyond kitty. Sounds like I might need to change my default system font or something but if kitty has support for these glyphs built in why is it showing incorrectly in kitty?

**Comment by kovidgoyal** [comment]
On Thu, Sep 19, 2024 at 10:25:30AM -0700, Tanner Wilcox wrote:

> I know this issue may span beyond kitty. Sounds like I might need to change my default system font or something but if kitty has support for these glyphs built in why is it showing incorrectly in kitty?

Because kitty only uses the builtin font if the glyphs are not available
in a system font. See th ekitty FAQ which tells you how to overcome this
precise issue.

**Reply by twiclo** [reply]
Which section in the FAQ? I just read the "I can't use my favorite font" section again and didn't find my answer. If it matters I'm using Source Code Pro. I selected it using `kitten choose-fonts`

**Reply by kovidgoyal** [reply, marked as answer]
symbol_map

**Reply by twiclo** [reply]
Adding that line to the config fixed it. Thanks
